swagger.yaml

```diff
@@ -4095,19 +7263,16 @@ definitions:
         type: string
         format: uuid
         description: Cluster ID on OCP system.
-      image_info:
-        $ref: '#/definitions/image_info'
-        x-go-custom-tag: gorm:"embedded;embedded_prefix:image_"
```

Comment:
This is a breaking change, right?
We will need versioned definitions as well.

Reply:
Yep that's a mistake, will fix tomorrow - thanks!

Reply:
Done